core/sup_tree_core/executor_pool/async_job_runner.ex

```diff
@@ -28,7 +28,7 @@ defmodule AntikytheraCore.ExecutorPool.AsyncJobRunner do
   alias AntikytheraCore.AsyncJob.Queue.JobKey
   alias AntikytheraCore.Context, as: CoreContext
   alias AntikytheraCore.GearLog.{Writer, ContextHelper}
-  require AntikytheraCore.Logger, as: L
+  alias AntikytheraCore.ExecutorPool.AsyncJobLog.Writer, as: CoreWriter
```

Comment:
I think that `JobLogWriter` is better than `CoreWriter` as for an alias name of `AntikytheraCore.ExecutorPool.AsyncJobLog.Writer`.

It is because that a reader may interpret `CoreWriter` as a writer module which is used to write to erlang.log.

Reply:
I fixed it.
https://github.com/access-company/antikythera/pull/93/commits/30cb654dc619bb43768a5c4df0da30ffe7670b14